packages/schematics/update/update/npm.ts

```diff
@@ -46,23 +44,22 @@ export function getNpmPackageJson(
   if (!maybeRequest) {
     const subject = new ReplaySubject<NpmRepositoryPackageJson>(1);
 
-    const protocolPackage = (fullUrl.protocol == 'https' ? https : http) as typeof http;
-    const request = protocolPackage.request(fullUrl, response => {
-      let data = '';
-      response.on('data', chunk => data += chunk);
-      response.on('end', () => {
-        try {
-          const json = JSON.parse(data);
-          json.requestedName = packageName;
-          subject.next(json as NpmRepositoryPackageJson);
-          subject.complete();
-        } catch (err) {
-          subject.error(err);
-        }
-      });
-      response.on('error', err => subject.error(err));
+    const client = new RegistryClient({
+
+    });
+    client.log.level = 'silent';
+    const params = {
+      timeout: 1000,
```

Comment:
Timeout needs to be larger.

Reply:
I'd say 30 seconds is fine. All requests are executed in parallel aynway.

Reply:
Done.